cdbot/cogs/admin.py

```diff
@@ -0,0 +1,51 @@
+import re
+
+from cbot.constants import NICKNAME_PATTERNS, PLACEHOLDER_NICKNAME
```

Comment:
```suggestion
from cdbot.constants import NICKNAME_PATTERNS, PLACEHOLDER_NICKNAME
```
Would cause an import error.